upgrade/capsule.py

```diff
@@ -26,6 +28,48 @@
 logger = logger()
 
 
+def satellite_capsule_setup(satellite_host, capsule_hosts, os_version,
+                            upgradable_capsule=True):
+    """
+    Setup all pre-requisites for user provided capsule
+
+    :param satellite_host: Satellite hostname to which the capsule registered
+    :param capsule_hosts: List of capsule which mapped with satellite host
+    :param os_version: The OS version onto which the capsule installed e.g: rhel6, rhel7
+    :param upgradable_capsule:Whether to setup capsule to be able to upgrade in future
+    :return: capsule_hosts
+    """
+    if os_version == 'rhel6':
+        baseurl = os.environ.get('RHEL6_CUSTOM_REPO')
+    elif os_version == 'rhel7':
+        baseurl = os.environ.get('RHEL7_CUSTOM_REPO')
+    else:
+        logger.warning('No OS Specified. Terminating..')
+        sys.exit(1)
+    non_responsive_host = list()
+    for cap_host in capsule_hosts:
+        if not host_pings(cap_host):
+            non_responsive_host.append(cap_host)
+        else:
+            execute(host_ssh_availability_check, cap_host)
+        execute(foreman_service_restart, host=cap_host)
```

Comment:
Do we restart the services on non_responsive_hosts too? Or should it be done for the responsive ones only?